### DISCUSSION TITLE
how to change color of part of title words in nicegui-timeline

### DISCUSSION THREAD

**Discussion by timedcy** [zauberzeug/nicegui#2763] [Q&A] [answered]
how to change color of part of title words in nicegui-timeline?
the html tags cannot work in title

**Comment by falkoschindler** [comment, marked as answer]
Hi @timedcy,

Do you mean something like this?
```py
with ui.timeline(side='right'):
    entry = ui.timeline_entry('This is the content of the timeline entry.', subtitle='March 25, 2024')
    with entry.add_slot('title'):
        ui.html('The <span style="color: #6495ED">Title</span>.')
```

<img width="307" alt="Screenshot 2024-03-25 at 11 55 20" src="https://github.com/zauberzeug/nicegui/assets/5767091/38fdf9a4-eb28-4360-9d0a-1d81bd2adf4e">

**Reply by timedcy** [reply]
Yes, that's it. Thx~
